### ISSUE TITLE
specs are hanging under master

### ISSUE THREAD

**Issue by colinsurprenant** [logstash-plugins/logstash-input-tcp#37]
intermittently the specs are just hanging under master. not sure if this is related to #36 ?

When they hang, it seems that no assertion is done at all:

```
Using Accessor#strict_set for specs
Run options: exclude {:redis=>true, :socket=>true, :performance=>true, :couchdb=>true, :elasticsearch=>true, :elasticsearch_secure=>true, :export_cypher=>true, :integration=>true, :windows=>true}

LogStash::Inputs::Tcp
```

**Comment by colinsurprenant** [comment]
probably related to https://github.com/elastic/logstash-devutils/issues/42

**Comment by colinsurprenant** [comment]
should be fixed with #38
